tests/.pylintrc

```diff
@@ -21,6 +21,7 @@ disable=bad-continuation,
         no-self-use,
         redefined-outer-name,
         too-few-public-methods,
+        too-many-arguments,
```

Comment:
Disabled this in the tests, as we've already done in other projects like LMS. One to backport into the cookie-cutter